TSC fails to compile on main

going through the readme and unable to get started. the commands in the readme failed here when trying to get the basics going this was a blocker.

```
% node -v
v23.5.0
% pnpm -v
9.15.2
```

```eliza-plugin-starter % pnpm build

> eliza-plugin-starter@0.1.0 build /Users/cole/Documents/GitHub/eliza-plugin-starter
> tsc && cp -R src/plugins dist/plugins

src/plugins/translation/provider/index.ts:2:62 - error TS2835: Relative import paths need explicit file extensions in ECMAScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Did you mean '../types.js'?

2 import { TranslationData, TranslationProviderResponse } from '../types';
                                                               ~~~~~~~~~~


Found 1 error in src/plugins/translation/provider/index.ts:2
```

--- a/src/plugins/translation/provider/index.ts
+++ b/src/plugins/translation/provider/index.ts
@@ -1,5 +1,5 @@
 import { Provider, IAgentRuntime, Memory, State } from '@ai16z/eliza';
-import { TranslationData, TranslationProviderResponse } from '../types';
+import { TranslationData, TranslationProviderResponse } from '../types.ts';
 
 // Simple in-memory cache
 const translationCache = new Map<string, TranslationData>();

